src/marketing/product-recommendations.md

```diff
@@ -21,7 +21,30 @@ This dashboard displays a table of previously configured recommendations (if any
 
 See [Recommendation Types]({% link marketing/prex-types.md %}) to learn about the available recommendation types in Magento.
 
-## Backup recommendations {#backup-recommendations}
+## The "Cold Start" problem
+
+The "Cold Start" problem refers to how much time a model needs to train before it can be considered high quality. In product recommendations, it translates to waiting for Adobe Sensei to train its machine learning models before deploying Recommendation units on your site.
+
+How long that takes depends on the following variables:
+
+- Higher traffic volume contributes to faster learning
+- Some recommendation types train faster than others
+- Some recommendation types do not require any training
+
```

Comment:
this seems like a fitting place for a reminder that due to complexity of this problem we suggest deploying data collection to prod first, then implementing the rest, so that models can train during development (link to our devdocs article)